hs_core/serialization.py

```diff
@@ -171,6 +172,8 @@ def create_resource_from_bag(bag_content_path, preserve_uuid=True):
 
     try:
         rm.write_metadata_to_resource(resource)
+        # Force bag files to be re-written
+        create_bag_files(resource)
```

Comment:
@selimnairb Got it. Have modified bag_ingest.py and posted it here: https://github.com/hydroshare/hydroshare/blob/access-control-manual-migration/migration-scripts/bag_ingest.py Please take a look and see whether it looks all right to you. @mjstealey can pull the updated bag_ingest.py for the next dry run.